README.md

```diff
@@ -93,14 +100,14 @@ The key words "MUST", "MUST NOT", "SHOULD", etc. are to be interpreted as descri
 
 ### Software requirements
 - Software requirements SHOULD be declared in a conda `environment.yml` file, including exact version numbers. Additionally, there MUST be a `Dockerfile` that containerizes the environment, or packages the software if conda is not available.
+- Docker containers MUST BE identified by their `sha256(Dockerfile + environment.yml)`. Like that, identical containers are implicitly re-used across subcommands.
```

Comment:
Maybe mentioning that subcommands will use the same container although each of them will have their own Dockerfile